docs/dev/migrating-to-v1.rst

```diff
@@ -333,6 +333,10 @@ has not been changed compared to the GridModel.
 | model.setup_grid_from_*(...) | model.grid_component.add_data_from_*(...) |
 +------------------------------+-------------------------------------------+
 
+TableComponent
+^^^^^^^^^^^^^^
+
+Same as Grid, but it's called Table component and has less methods.
```

Comment:
Maybe add that this used to be self.tables.
I thought I made you start a list of added components as well and their meaning (is it here still a dict of pandas.DataFrame for non-geospatial tabular data?)

Reply:
On top I wonder if we don't keep the plural here: ``tables: TablesComponent`` to mention that this component actually contains several tables/pd.Dataframe. Maybe we will/can add a singular one ``table: TableComponent`` for single and direct self.data: pd.Dataframe.
Same question for the current ``geoms``.
@DirkEilander what do you think?

Reply:
I like the table, I think that's a good idea so I'll add that. I'm not a fan of adding singular versions on top of the plural, but I don't really care much about which is will be, so I'm happy to defer there

Reply:
I've added the table so what remains is the plural/singular. @hboisgon @DirkEilander what do you think?

Reply:
then we will keep the plural

Reply:
Dirk agreed to keep the plural as well